projects/assets/src/translations/en/my-account.ts

```diff
@@ -85,6 +85,14 @@ export const myAccount = {
     noOrders: 'We have no order records for this account.',
     startShopping: 'Start Shopping',
     sortByMostRecent: 'Sort by Most recent',
+    replenishmentOrderHistory: 'Replenishment Order History',
+    replenishmentOrderId: 'Replenishment #',
+    purchaseOrderNumber: 'PO #',
+    startOn: 'Start On',
+    frequency: 'Frequency',
+    nextOrderDate: 'Next Order Date',
+    estTotal: 'Est Total',
```

Comment:
ok, there was no design, so the only guidance on col names was wires.

Reply:
when you comment, you should leave it open for me to resolve, so that I can answer back

* I believe we spoke about this already in our direct chat, where you asked if we should re-use, which I believe I said yes

Reply:
that was for something that is exactly the same, but `est total` is something new in the file. either way, there will be one off cases where you do or do not follow the wires. in this case, we are not following the wires, hence the issue was resolved.